packages/documentation-framework/templates/mdx.js

```diff
@@ -25,60 +41,107 @@ const MDXChildTemplate = ({
-    }
+    };
     ensureID(toc);
   }
 
-  const isComponentCodeDocs = ['react', 'react-demos', 'html', 'html-demos', 'react-templates'].includes(source);
+  const isComponentCodeDocs = [
+    'react',
+    'react-demos',
+    'html',
+    'html-demos',
+    'react-templates',
+  ].includes(source);
 
-  const InlineAlerts = (optIn || beta || deprecated || source === 'react-deprecated' || source === 'html-deprecated' || template || source === 'react-template') && (
+  const InlineAlerts = (optIn ||
+    beta ||
+    deprecated ||
+    source === 'react-deprecated' ||
+    source === 'html-deprecated' ||
+    // TODO: remove hardcoded Tile when Core PR merges
+    // https://github.com/patternfly/patternfly/pull/7178
+    id === 'Tile' ||
+    template ||
+    source === 'react-template') && (
     <StackItem>
-      {optIn && (
-        <InlineAlert title="Opt-in feature">
-          {optIn}
-        </InlineAlert>
-      )}
+      {optIn && <InlineAlert title="Opt-in feature">{optIn}</InlineAlert>}
       {beta && (
         <InlineAlert title="Beta feature">
-          This beta component is currently under review and is still open for further evolution. It is available for use in product. Beta components are considered for promotion on a quarterly basis. Please join in and give us your feedback or submit any questions on the <a href="https://forum.patternfly.org/">PatternFly forum</a> or via <a href="//slack.patternfly.org/" target="_blank" rel="noopener noreferrer">Slack</a>. To learn more about the process, visit our <Link to="/get-started/about-patternfly#beta-components">about page</Link> or our <a href="https://github.com/patternfly/patternfly-org/tree/main/beta-component-promotion">Beta components</a> page on GitHub.
+          This beta component is currently under review and is still open for
+          further evolution. It is available for use in product. Beta components
+          are considered for promotion on a quarterly basis. Please join in and
+          give us your feedback or submit any questions on the{' '}
+          <a href="https://forum.patternfly.org/">PatternFly forum</a> or via{' '}
+          <a
+            href="//slack.patternfly.org/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Slack
+          </a>
+          . To learn more about the process, visit our{' '}
+          <Link to="/get-started/about-patternfly#beta-components">
+            about page
+          </Link>{' '}
+          or our{' '}
+          <a href="https://github.com/patternfly/patternfly-org/tree/main/beta-component-promotion">
+            Beta components
+          </a>{' '}
+          page on GitHub.
         </InlineAlert>
       )}
-      {(deprecated || source === 'react-deprecated' || source === 'html-deprecated') && (
+      {(deprecated ||
+        source === 'react-deprecated' ||
+        source === 'html-deprecated' ||
+        // TODO: remove hardcoded Tile when Core PR merges
```

Comment:
Can we open a follow up issues so we don't forget to do this?

Reply:
Thank you!  Created here - https://github.com/patternfly/patternfly-org/issues/4343